### ISSUE TITLE
High sampling rate

### ISSUE THREAD

**Issue by jpsemery** [PhilJay/MPAndroidChart#1118]
Hi
This is a question more than an issue.
I'm writing a data acquisition application. I'm wondering what kind of sampling rate a MPAndroidChart real-time simple line graph can handle. I'm looking at up to 1024 samples/secs.

I understand that this will vary with devices and I'm targetting fairly high end phones/tablets. I'm currently developing using a HTC one M8

Thanks for you help.

**Comment by PhilJay** [comment]
If you want to refresh 1024 times per second, this will be an issue. But since the human eye cannot capture more than ~24 frames per second, it makes no sense to refresh any more often than that.

**Comment by jpsemery** [comment]
You were a bit quick to close this.
It's not about refresh rate or perception, it's about data representation. So this comes down to the number of pixels on the x axis available to represent the data samples. 

So let me rephrase my question. Can MPAndroid chart refresh a simple line graph of 1920 (screen width) points 24 times a second?

Thanks again

**Comment by PhilJay** [comment]
This will depend on the device. If you keep the graph simple (e.g. like in the example app - "Performance Linechart") up to 30.000 data points can be rendered. This number will decrease when u use specific styling, such as cubic-lines, filled-lines, ...
